src/pages/ShoppingCart.page.js

```diff
@@ -1,5 +1,6 @@
 import { BaseSwagLabPage } from './BaseSwagLab.page';
 
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Lets move this to the eslint config file

Reply:
done